uaclient/apt.py

```diff
@@ -320,7 +320,7 @@ def clean_apt_files(*, _entitlements=None):
         from uaclient import entitlements as _entitlements
 
     for ent_cls in _entitlements.ENTITLEMENT_CLASSES:
-        if not isinstance(ent_cls, RepoEntitlement):
+        if not isinstance(ent_cls(), RepoEntitlement):
```

Comment:
I think this should be issubclass test instead of doing the work to instantiate the subclass
```suggestion
        if not issubclass(ent_cls, RepoEntitlement):
```